docs/requirements.txt

```diff
@@ -3,3 +3,4 @@ sphinx-prompt==1.1.0
 sphinx-tabs==1.1.10
 sphinx-rtd-theme==0.4.3
 sphinx-notfound-page==0.3
+git+https://github.com/rtfd/readthedocs-sphinx-search@master
```

Comment:
You should be able to have it install in the `conf.py` using the local path, so it always uses the version for the build. Perhaps something like this: `https://github.com/rtfd/sphinx-autoapi/blob/master/requirements.txt#L1`

Reply:
Similar method: https://github.com/rtfd/sphinx-notfound-page/blob/master/.readthedocs.yml#L6-L7

Reply:
I have taken reference from not-found-page... this is already done.
I am removing this line then - `git+https://github.com/rtfd/readthedocs-sphinx-search@master`

Reply:
Didn't remember it earlier.. !!